src/interfaces/IOptionSettlementEngine.sol

```diff
@@ -200,6 +200,13 @@ interface IOptionSettlementEngine {
      */
     error InvalidAssets(address asset1, address asset2);
 
+    /**
+     * @notice The supplied underlying and exercise amounts are too small.
+     * @param underlyingAmount The amount of the underlying asset.
+     * @param exerciseAmount The amount of the exercise asset.
+     */
+    error InvalidAssetAmounts(uint256 underlyingAmount, uint256 exerciseAmount);
```

Comment:
Nit — slight pref for `InvalidAmounts`, better symmetry with `InvalidAssets` imo but I can see either way.

Reply:
n/a, going to be removing the error type